### DISCUSSION TITLE
Unified Alerting: Allow for default data source selection

### DISCUSSION THREAD

**Discussion by jpdstan** [grafana/grafana#38798] [Alerting]
# Problem
It would be nice if the Unified Alerting UI did not try to load rules in from every single datasource, as we have many datasources setup where we don't necessarily want to pull from. For example, we have a backup Prometheus install that we leave running that hosts the same rules as our default Prometheus install. This means our UI will have duplicate rules until we select the desired default datasource.

Moreover, datasources that do not have the proper routing setup result in errors like this:

![image](https://user-images.githubusercontent.com/5893781/131715715-56c1d05e-b415-4890-b4f3-45f5bbda2f48.png)

# Solution
Allow for some UI where we can enumerate the default datasource(s). Even having a "Save" function like in Dashboards where you can check the "Save current datasource as default" would be sufficient (though that would allow for only one datasource).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
